src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml.cs

```diff
@@ -119,13 +120,29 @@ private void RunAutoCCA(Bitmap bitmap)
 
             if (pair == null)
             {
+                Logger.PublishTelemetryEvent(
+                    TelemetryAction.ColorContrast_AutoDetect,
```

Comment:
Can we move the call to `Logger.PublishTelemetryEvent` to before the guard on line 123? We could use a ternary operator to easily determine what string to use for `TelemetryProperty.Confidence`

Reply:
It looks like `confidence` is automatically set to `None`, so no need for two checks. Thanks!